### PR TITLE
Packaging files

### PR DESCRIPTION
**Pull request by jyeray**

### CODE REVIEW

File: package/yast-update-alternatives.spec [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+Name:           yast2-update-alternatives
+Version:        0.1
```
Comment (jreidinger) [inline package/yast-update-alternatives.spec]: I suggest to follow yast versioning, which is now `3.1.X` so in your case `3.1.0` to be consistent.

File: package/yast-update-alternatives.spec [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Summary:        YaST2 - Manage Update-alternatives switching
+Url:            https://github.com/yast/yast-update-alternatives
+Group:          System/Yast
+Source0:        %{name}-%{version}.tar.bz2
+BuildRequires:  yast2
+BuildRequires:  yast2-devtools
+BuildRequires:  yast-ruby-bindings
+# For install
+BuildRequires:  rubygem(yast-rake)
+# For testdd
```
Comment (jreidinger) [inline package/yast-update-alternatives.spec]: typo `test`

**Comment by jreidinger** [comment]
spec looks quite good, also please add changes file, you can simply run there `osc vc` where you write your initial commit msg. I also create initial pkg in build service. So you can verify it install as it should via running `rake osc:build`

**Comment by jreidinger** [comment]
good job :+1:  LGTM

**Comment by jyeray** [comment]
I have some problems running `rake osc:build`, so I make some little changes. 
Now it generates the RPM and it works. But in the output of `rake osc:build` show this:

```
The buildroot was: /var/tmp/build-root-openSUSE_Factory
rm -rf YaST:Head
cleaning
rake aborted!
Command failed with status (1): [osc build --no-verify --release=1 --root=/...]

Tasks: TOP => osc:build
(See full trace by running task with --trace)
```

I don’t know what is happening.

**Pull request merged**